syntax = "proto3";
package protocol;

import "center_enum.proto";
import "center_struct.proto";

message CenterServerHandshake {
  int32 version = 1;
}

message CenterClientRegisterRequest {
  string name = 1;
  string ip = 2;
  int32 port = 3;
  ServerType type = 4;
}

message CenterServerRegisterResponse {
  ServerRegisterResult result = 1;
}